forPublication/vsm.md

```diff
@@ -1,42 +1,36 @@
-![](parachute.jpg)
-
+![](../parachute.jpg)
 #### You perceive your time to market for new features to be slow and want to speed up
-You experience that things are slow, and it's hard to finish anything within an iteration. You've tried to deliver 
-smaller increments, but it is still a struggle to complete these in a reasonable amount of time. 
-
-One of the reasons might be that wasteful activities and unnecessary handovers are hiding in your
-development process. 
-
+You experience that things are slow, and it's hard to finish anything within an iteration. 
+You've tried to deliver smaller increments, but it is still a struggle to complete these in a reasonable amount of time. 
+One of the reasons might be that wasteful activities and unnecessary handovers are hiding in your development process. 
 #### A Value Stream Map (VSM) can help you uncover this kinds of waste in your software delivery process
-
-Value Stream Mapping is a form of lean-management and was one of the foundations that made the Toyota Production 
-System. It maps the series of events that take a product or service from the beginning of the specific process (inception) 
-until it reaches the customer. 
-<br/><br/> Even though it stems from a manufacturing world, it can be applied to software as well.
-There it is an extremely helpful technique because of the high cost of of context switching in 
-knowledge work and software development specifically (link to blog the hidden cost of context switching) 
- 
+Value Stream Mapping is a form of lean-management and was one of the foundations that made the Toyota Production System.
```

Comment:
`was one of the foundations that made the Toyota Production System` suggests that it made the Toyota Production System **something**. I would probably reword it to `was at the foundation of the Toyota Production System`